BedrockServer.cpp

```diff
@@ -1404,16 +1403,20 @@ bool BedrockServer::shutdownComplete() {
 }
 
 void BedrockServer::prePoll(fd_map& fdm) {
-    STCPServer::prePoll(fdm);
+    // Add all our ports. There are no sockets directly managed here.
```

Comment:
All the actual sockets are now handled in socket threads, so all we need to care about is our ports.